python/lsst/meas/algorithms/subtractBackground.py

```diff
@@ -274,6 +283,20 @@ def _addStats(self, exposure, background, statsKeys=None):
         meta.addDouble(mnkey, bgmean)
         meta.addDouble(varkey, bgvar)
 
+    def filterSuperPixels(self, bbox, background):
+        """Remove outliers from the binned background model.
+
+        Parameters
+        ----------
+        bbox : `lsst.geom.Box2I`
+            Bounding box of the original image.
+        background : `lsst.afw.math.BackgroundMI`
+            Fit and binned background image, which will be modified in place.
+        """
+        statsImg = background.getStatsImage()
+        statsImg.image.array = median_filter(statsImg.image.array, mode='reflect', size=3)
```

Comment:
I'd certainly like to see the `size=3` as a configurable here. I'm less concerned of the need to make `mode` configurable though (I think `reflect` is a good one-size-fits-all here).